site/src/pages/components/focusgroup.explainer.mdx

```diff
@@ -270,6 +272,22 @@ from "selection/toggle" state.
 A future extension of this proposal may allow focus entering a focusgroup to jump to the currently
 toggled (or the toggle with the highest value) if any toggles are set in the focusgroup.
 
+Focusgroup definitions apply by default across Shadow DOM boundaries in order to make it easy for component
```

Comment:
again I would remove "by default" as this is showing how people can leverage standards that exist to make it so that their shadow children are not focusable.

Reply:
Done.